src/workers/notifications/warmIntroNotification.ts

```diff
@@ -0,0 +1,49 @@
+import { TypedNotificationWorker } from '../worker';
+import { NotificationType } from '../../notifications/common';
+import { logger } from '../../logger';
+import { OpportunityJob } from '../../entity/opportunities/OpportunityJob';
+import { OpportunityUserType } from '../../entity/opportunities/types';
+import { WarmIntro } from '@dailydotdev/schema';
+
+export const warmIntroNotification: TypedNotificationWorker<'gondul.v1.warm-intro-generated'> =
+  {
+    subscription: 'api.warm-intro-notification',
+    handler: async ({ userId, opportunityId, description }, con) => {
```

Comment:
```suggestion
    subscription: 'api.warm-intro-notification',
    parseMessage: (message) => WarmIntro.fromBinary(message.data),
    handler: async ({ userId, opportunityId, description }, con) => {
```

and remove the one below

Reply:
linter will warn you if you don't set parseMessage